src/avatar.tsx

```diff
@@ -34,6 +34,8 @@ type AvatarProps = {
     badge?: boolean | number;
     'aria-label'?: string;
     dataAttributes?: DataAttributes;
+    border?: boolean;
+    borderColor?: 'border' | 'borderLow' | 'borderHigh' | 'borderSelected';
```

Comment:
these colors are not defined in the specs, why did you decide to use these colors?

I think is better to let it open, because could have some cases that you need to use the same color of the background

Reply:
yes, borderLow could works in most of cases

Reply:
Yaio, nós estamos utilizando os Tokens de Borda nas bordas conforme padrões Mística. Existem Tokens para Bordas e seguimos as definições, podemos abrir se vocês acharem que é válido, mas aí será um uso muito amplo e despadronizado.

Vocês querem deixar aberto? Se sim nós o faremos, mas ao meu ver é perigoso.

Reply:
It is ok, to keep aligned with Avatar border specs that Vivo design team defined, the best option is remove the possibility of change border color. If in the future is need it, we will evolve to allow that.

Reply:
yes, please, dont make this configurable.
We only want a boolean prop called `border`, when provided, the component will apply a border of `1px solid ${skinVars.colors.borderLow}`